Classes/Interface/Hook/Registration.php

```diff
@@ -12,10 +12,15 @@
      *
      * @param \Tx_Seminars_OldModel_Registration $registration
      * @param \Tx_Oelib_Template $emailTemplate
+     * @param \Tx_Oelib_Mail $eMailNotification
      *
      * @return void
      */
-    public function modifyOrganizerNotificationEmail(\Tx_Seminars_OldModel_Registration $registration, \Tx_Oelib_Template $emailTemplate);
+    public function modifyOrganizerNotificationEmail(
+        \Tx_Seminars_OldModel_Registration $registration,
+        \Tx_Oelib_Template $emailTemplate,
+        \Tx_Oelib_Mail $eMailNotification
+    );
```

Comment:
The new hook should be called after the body for the email has been set, i.e., it should not get `$emailTemplate` or `$registration` anymore. Maybe a good name would be `postProcess*Email`. What do you think?

Reply:
> Using "current best practices" goes over "being consistent with old code" here.)

So true

The name sounds great. I think it could be useful to keep the $registration e.g. if someone extends the Event model $registration->seminar with a custom sender name field. Don´t know how realistic that is.

Reply:
Yes, the new hook should also get the registration.